File: Data Structures & Algorithms/two-integer-sum/submission-1.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        int a=0, b=0, total = 0;
        bool found = false;
        for(int i = 0; i < nums.Length; i++)
        {
            for(int j = i + 1; j < nums.Length ; j++)
            {
                total = nums[i] + nums[j];
                if (total == target) {
                    a = i;
                    b = j;
                    found = true;
                    break;

                }
               
            }
            if (found)
            break;
       
        }
        
        return new int[] {a, b};

    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        #Brute Force solution, TC = 
        if not nums:
            return 0
        
        longest = 1
        for i in range(len(nums)):
            current = nums[i] + 1
            count = 1
            while True:
                found = False
                for j in range(len(nums)):
                    if current == nums[j]:
                        count += 1
                        current += 1
                        found = True
                        break

                if not found:
                    break

            
            longest = max(longest,count)

        return longest

